OracleGoldenGate/21c/README.md

```diff
@@ -0,0 +1,151 @@
+
+All parameters are optional, so the following command will work, too:
+
+```sh
+$ docker run oracle/goldengate:21.3.0.0.0
+----------------------------------------------------------------------------------
+--  Password for OGG administrative user 'oggadmin' is 'XU2k7cMastmt-DJKs'
+----------------------------------------------------------------------------------
+...
+```
+
+See the following sections for additional details.
+
+### Administrative Account Password
+
+On the first startup of the container, a random password will be generated for the Oracle GoldenGate administrative user if not provided by the `OGG_ADMIN_PWD` environment variable. You can find this password at the start of the Docker container log:
+
+```sh
+$ docker logs <container name> | head -3
+----------------------------------------------------------------------------------
+--  Password for OGG administrative user 'oggadmin' is 'ujX7sqQ430G9-xSlr'
+----------------------------------------------------------------------------------
+```
+
+### SSL Certificate
+
+When bringing your own SSL certificate to an Oracle GoldenGate Docker container, two files are needed:
+
+1. `ogg.key` - The private key for the SSL certificate.
+1. `ogg.pem` - The SSL leaf certificate, and a full certificate trust chain
+
+If these files are located in a directory called `cert`, they can be used in the GoldenGate Docker container with a volume mount as shown here:
+
+```sh
+$ docker run -v ${PWD}/cert:/etc/nginx/cert:ro -p 8443:443 ogg-21c
+...
+```
+
+The certificate file, `ogg.pem`, must contain a full certificate chain starting with the leaf certificate, and followed by all other certificates in the Certificate Authority chain.
+
+```pem
+-----BEGIN CERTIFICATE-----
+MIIFBTCCA+2gAwIBAgISBJSzNXE+Ha5eDw76N5lgHhTpMA0GCSqGSIb3DQEBCwUA
+MEoxCzAJBgNVBAYTAlVTMRYwFAYDVQQKEw1MZXQncyBFbmNyeXB0MSMwIQYDVQQD
+...
+dr7wTE+AQwcOLAGjIvFOL7GK8JrhKvuFvnSoys/1O2CK3vVhBgS+mEF6D+QjIGTv
+VC01LCPT51q58INy4RtDBPSqlJwrzz+pOOWd5rBWhu2UPktVHz3AtYE=
+-----END CERTIFICATE-----
+-----BEGIN CERTIFICATE-----
+MIIEkjCCA3qgAwIBAgIQCgFBQgAAAVOFc2oLheynCDANBgkqhkiG9w0BAQsFADA/
+MSQwIgYDVQQKExtEaWdpdGFsIFNpZ25hdHVyZSBUcnVzdCBDby4xFzAVBgNVBAMT
+...
+PfZ+G6Z6h7mjem0Y+iWlkYcV4PIWL1iwBi8saCbGS5jN2p8M+X+Q7UNKEkROb3N6
+KOqkqm57TH2H3eDJAkSnh6/DNFu0Qg==
+-----END CERTIFICATE-----
+```
+
+### Running the Administration Client
+
+The **Administration Client** utility can be run with this command:
+
+```sh
+$ docker exec -ti --user ogg <container name> adminclient
+Oracle GoldenGate Administration Client for Oracle
+Version 21.3.0.0.0 ...
+```
+
+## Known Issues
+
+None
+
+## License
+
+All scripts and files hosted in this project and GitHub [docker-images/OracleGoldenGate](../) repository required to build the Docker images are, unless otherwise noted, released under the Universal Permissive License (UPL), Version 1.0.  See [LICENSE](../LICENSE) for details.
```

Comment:
Replace Docker with container and link to the `LICENSE` file in the root of the repo.

Reply:
`LICENSE` is already referenced here.  Am I missing something?

Reply:
I changed the references to `LICENSE` and removed that file from the `OracleGoldenGate` directory.